src/core/QCheck.mli

```diff
@@ -1203,6 +1203,7 @@ val numeral_char : char arbitrary
 (** Uniformly distributed over ['0'..'9']. *)
 
 val string_gen_of_size : int Gen.t -> char Gen.t -> string arbitrary
+(** Generates strings of exactly length [int Gen.t]. *)
```

Comment:
Thanks - and well spotted!

Since the size of the resulting strings vary with the output of the first argument (a "pure" Gen.t generator),
perhaps the documentation should be more along the lines of `Gen.string_size` that it calls underneath? 
```ocaml
  val string_size : ?gen:char t -> int t -> string t
  (** Builds a string generator from a (non-negative) size generator.
      Accepts an optional character generator (the default is {!char}). *)
```

Reply:
Agreed that's a better description.